src/simulation/dynamics/spacecraft/_UnitTest/test_spacecraft.py

```diff
@@ -1090,6 +1092,10 @@ def scAccumDV():
             testFailCount += 1
             testMessages.append("FAILED: Spacecraft Point B Accumulated DV test failed pos unit test")
 
+        if not unitTestSupport.isArrayEqual(dataAccumDV_CN_N[i+1],truth_dataAccumDV_CN_N,3,accuracy):
```

Comment:
:large_blue_circle: If we had the time it would be beneficial to move away from the `testFailCount` approach but I'll let you decide if you have the time to make the change.

Reply:
I will leave it as it is for the sake of time. However, in the meantime I'll educate myself on Python's test functions to use them in the future.